modules/rosa-sts-associating-your-aws-account.adoc

```diff
@@ -40,7 +40,7 @@ ifndef::rosa-hcp[]
 endif::rosa-hcp[]
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
+* You have installed and configured the latest {product-title} (ROSA) CLI (`rosa`) on your installation host.
```

Comment:
just ROSA CLI here, the product title has already been expanded in the {hcp-title} or {product-title} above

Reply:
Maybe, "ROSA CLI (`rosa`)" ? As this is the first instance.